website/blog/2021-02-22-7.13.0.md

```diff
@@ -240,16 +240,16 @@ Babel 7.12.0 two new Flow features:
 
 ### TypeScript 4.2 support ([#12628](https://github.com/babel/babel/pull/12628))
 
-TypeScript 4.2 is currently in beta (TODO: Feb 9th it will be RC), and it supports some new syntax features such as abstract constructor signatures.
+[TypeScript 4.2](https://devblogs.microsoft.com/typescript/announcing-typescript-4-2-rc/) is currently RC, and it supports some new syntax features such as abstract constructor signatures.
```

Comment:
```suggestion
[TypeScript 4.2](https://devblogs.microsoft.com/typescript/announcing-typescript-4-2-rc/) is currently RC (it will become stable on the 23rd, but we beated them by one day!) , and it supports some new syntax features such as abstract constructor signatures.
```

wdyt, assuming that we actually release on Monday 22nd? :joy: